sensors/accelerometer/accelerometer.ino

```diff
@@ -0,0 +1,24 @@
+#include "accelerometer.h"
+#include <SPI.h>
+// https://github.com/sparkfun/SparkFun_LSM9DS1_Arduino_Library/archive/master.zip
+#include <SparkFunLSM9DS1.h>
+
+LSM9DS1 accelerometer;
+
+Accelerometer::Accelerometer(const int &pin): Cartesian_Sensor(&pin) {}
```

Comment:
Definitely have to initialize our member variables to something. If read_sensor_value goes into the first if statement and then the user attempts to print the sensor value, we would get an undefined error.